mod-infra/src/main/scala/at/forsyte/apalache/infra/log/LogbackConfigurator.scala

```diff
@@ -24,21 +24,23 @@ class LogbackConfigurator(runDir: Option[Path], customRunDir: Option[Path]) exte
     setContext(loggerContext)
     runDir match {
       case Some(_) => configure(loggerContext)
-      case None    => configureSilent(loggerContext)
+      case None    => configureConsoleOnly(loggerContext)
```

Comment:
Are you sure this doesn't mess with tests?

Where else do we end up here without a `runDir`?

Reply:
It does not mess with the tests. This setup is only needed before the actual configuration is read. If something happens when reading the configuration, e.g., parsing the tuning parameters fails, we should be able to see the error messages.